www/caddy/src/opnsense/mvc/app/models/OPNsense/Caddy/Caddy.php

```diff
@@ -107,6 +108,52 @@ private function checkSubdomainsAgainstDomains($subdomains, $domains, $messages)
         }
     }
 
+    // 4. Get the current OPNsense WebGUI ports
+    private function getWebGuiPorts() {
+        $webgui = Config::getInstance()->object()->system->webgui ?? null;
+        $webGuiPorts = [];
+
+        // Only add ports to array if no specific interfaces for the WebGUI are set
+        if (!empty($webgui) && empty((string)$webgui->interfaces)) {
+            // Add port 443 if no specific port is set, otherwise set custom webgui port
+            if (empty($webgui->port)) {
```

Comment:
you probably want to ditch the if here 🙂 (non empty means no port in the list

Reply:
I totally missed that, thank you.